src/API/Logs/EventLoggerInterface.php

```diff
@@ -13,7 +13,7 @@ interface EventLoggerInterface
 {
     public function emit(
         string $name,
-        mixed $payload = null,
+        mixed $body = null,
```

Comment:
This is technically a breaking change if you are using named parameters; should we have any concerns around this?

Reply:
Only minor concerns. Since [that part of the spec is still in-development](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md), changes should be expected. In fact, the whole logs signal seems to still be in-development/experimental.